src/chai-form-configurator.ts

```diff
@@ -0,0 +1,414 @@
+import { LitElement, html, css } from 'lit';
+import { property, customElement } from 'lit/decorators.js';
+import { ChaiForm } from './chai-form';
+
+@customElement('chai-form-configurator')
+export class ChaiFormConfigurator extends LitElement {
+  @property({ type: String }) targetForm: string = '';
+
+  static override styles = css`
+   
+    .form-configurator-container
+    {
+      color: black;
+    padding: 20px;
+    background-color: #fbf7f1;
+    border-radius: 20px;
+    text-shadow: none;
+    font-weight: bolder;
+    font-size: 14px;
+    }
+
+    .row
+    {
+      margin-bottom:15px;
+      font-family:sans-serif;
+    }
+    .form-configurator-header-message
+    {
+      margin-bottom: 34px;
+    font-weight: bold;
+    }
+  `;
+
+  constructor() {
+    super();
+    this.targetForm = ''; 
+  }
+
+  @property({ type: String }) selectedVisibilityValue: string = '';
+
+
+  toggleChaiFormSlots(event : Event ,  componentName : string)
+  {
+
+    
+     const target = document.getElementById(this.targetForm) as ChaiForm; 
+        if (!target) return;
+
+            
+
+          const checkbox = event.target as HTMLInputElement; 
+        if(!checkbox) return;
+
+        if(componentName == 'chai-name')
+        {
+          target.isCreateName = checkbox.checked;
+        }else if(componentName == 'chai-email')
+        {
+          target.isCreateEmail = checkbox.checked;
+        }else if(componentName == 'chai-address')
+        {
+          target.isCreateAddress = checkbox.checked;
+        }else if(componentName == 'chai-phone')
+        {
+          target.isCreatePhone = checkbox.checked;
+        }
+  }
+
+
+  applyChaiStyle(event: Event , cssVaraible:string , px:string)
+  {
+    console.log(event,cssVaraible);
+      const target = document.getElementById(this.targetForm) as ChaiForm; 
+        if (!target) return;
+        const input = event.target as HTMLInputElement;
+        target.style.setProperty(cssVaraible , input.value + px);
```

Comment:
This is where you should be writing a CSS snippet (that can render in a `<style>` element) instead of setting individual elements' properties. You made it harder on yourself and harder on the person trying to use this.